### DISCUSSION TITLE
eks 제외 올려야할 것들

### DISCUSSION THREAD

**Discussion by gimsoomin** [cloud2scape/.github#31] [cloud-Q&A]
---
@ssedogdoom TA님 eks 제외하고 cd나 obserbility 쪽에서 올려야 할 것들 알려주세요

**Comment by ssedogdoom** [comment]
ArgoCD 구축해주세요(DevOps repository에 예시 나와있습니다.
그 외에 더 필요한 것은 말씀드리겠습니다
